receiver/hostmetricsreceiver/internal/scraper.go

```diff
@@ -33,4 +33,13 @@ type ScraperFactory interface {
 
 // Config is the configuration of a scraper.
 type Config interface {
+	SetRootPath(rootPath string)
+}
+
+type ParentConfigSetter struct {
```

Comment:
Setter doesn't seem like a good name for a struct
```suggestion
type ScraperConfig struct {
```

Reply:
Done